rust/tvm-sys/Cargo.toml

```diff
@@ -32,4 +32,4 @@ ndarray = "0.12"
 enumn = "^0.1"
 
 [build-dependencies]
-bindgen = "0.51"
+bindgen = { version="0.51", default-features=false }
```

Comment:
I think @kazum wanted this on for building the WASM backend, can you confirm?

Reply:
The default-features of `bindgen` pulls in `clap` which has a pretty big (albeit `build-dep`) dependency load. So it'd be good to trim if possible.

Reply:
Thanks for following up @kazum all the big changes to Rust should be done. I will be sending a bunch of doc updates in coming weeks after I finish landing new Relay error reporting. Max and I have been using the bindings at OctoML for some experiments.